lib/preamble.dart

```diff
@@ -1,7 +1,16 @@
 library node_preamble;
 
 var _NODE_PREAMBLE = r"""
-  global.location = { href: "file://" + process.cwd() + "/" };
+  // TODO: This isn't really a correct transformation. For example, it will fail
+  // for paths that contain characters that need to be escaped in URLs. Once
+  // dart-lang/sdk#27979 is fixed, it should be possible to make it better.
+  global.location = {
+    href: "file://" + (function() {
+      var cwd = process.cwd();
+      if (process.platform != 'win32') return cwd;
+      return '/' + cwd.replace('\\', '/');
+    }) + "/"
```

Comment:
I'm guessing you meant to put a () here? I've fixed this on my own, but I wanted to let you know since you temporarily forked it.